src/eu/janinko/games/backgammon/Stone.java

```diff
@@ -1,42 +1,63 @@
 package eu.janinko.games.backgammon;
 
-import eu.janinko.games.backgammon.Stone.Color;
-
+/**
+ * Model of a Stone for the game of Backgammon
+ * <p>
+ * <b> To Use: </b> <br>
+ * Utilize the Stone object references Stone.WHITE, Stone.BLACK, & Stone.NONE to
+ * represent White, Black, and no reference stones respectively
+ * </p>
+ */
 public class Stone {
+	/**
+	 * Stone Color representation for a game of Backgammon
+	 * <p>
+	 * As in the classic game of Backgammon stones are in colors WHITE or BLACK,
+	 * there Color of NONE represents no color
+	 * </p>
+	 */
 	public enum Color {
-		WHITE,
-		BLACK,
-		NONE
+		WHITE, BLACK, NONE
 	}
+
+	// Stone references to be used throughout the application
 	public static Stone WHITE = new Stone(Color.WHITE);
 	public static Stone BLACK = new Stone(Color.BLACK);
 	public static Stone NONE = new Stone(Color.NONE);
-	
+
+	// Color of the Stone
 	private Color color;
-	
-	
-	public Stone(Color color) {
-		this.color = color;
-	}
 
-	public Stone(boolean color) {
-		this.color = (color?Color.WHITE:Color.BLACK);
-	}
-	
-	public Stone() {
-		this.color = Color.NONE;
+	/**
+	 * Constructor for a Stone instance
+	 * 
+	 * @param color
+	 *            The Color associated with the Stone
+	 * @see Color
+	 */
+	private Stone(Color color) {
+		if (color == null) {
+			// null is represented by a non-existing piece or NONE
+			this.color = Color.NONE;
+		} else {
+			// Set the color of this to the requested color
+			this.color = color;
+		}
 	}
 
+	/**
+	 * Fetches the color of the Stone
+	 * 
+	 * @return Color enumeration value for the stones color
+	 * @see Color
+	 */
 	public Color getColor() {
 		return color;
 	}
 
-	public void setColor(Color color) {
-		this.color = color;
-	}
-
 	@Override
 	public int hashCode() {
+		// Calculates a hash code value specific to each Stone using a low prime
```

Comment:
``hashCode`` is common Java method, I don't think it's necessary to have it commented when the common implementation is used and it uses obvious fields.